packages/graphile-build-pg/src/plugins/PgIntrospectionPlugin.js

```diff
@@ -185,7 +186,10 @@ export default (async function PgIntrospectionPlugin(
             versionResult.rows[0].server_version_num,
             10
           );
-          const introspectionQuery = makeIntrospectionQuery(serverVersionNum);
+          const introspectionQuery = makeIntrospectionQuery(
+            serverVersionNum,
+            pgLegacyFunctionsOnly
```

Comment:
I feel like this list of options could get quite big quite fast; should we make it an object instead?

Reply:
I went back and forth.. but taking a fresh look, I think it should be an options object. I'll make the change.